index.html

```diff
@@ -0,0 +1,304 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0" />
+    <title>Nova-website</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;700&display=swap"
+      rel="stylesheet" />
+    <link
+      rel="stylesheet"
+      href="./styles/reset.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.2/css/all.min.css" />
+    <link
+      rel="stylesheet"
+      href="./styles/style.css" />
+  </head>
+  <body>
+    <div class="background-image">
+      <header class="header row">
+        <div class="header__container container">
+          <a
+            href="#"
+            class="header__logo">
+            <img
+              class="header__logo-img"
+              src="./images/logo.svg"
+              alt="nova_logo" />
+          </a>
+          <nav class="header__nav nav">
+            <ul class="nav__list">
+              <li class="nav__item">
+                <a
+                  class="nav__link"
+                  href="./#team"
+                  >The Team</a
+                >
+              </li>
+              <li class="nav__item">
+                <a
+                  class="nav__link"
+                  href="./#pricing"
+                  >Pricing</a
+                >
+              </li>
+              <li class="nav__item">
+                <a
+                  class="nav__link"
+                  href="./#features"
+                  >Features</a
+                >
+              </li>
+              <li class="nav__item">
+                <button class="nav__button btn btn-blue">Download FREE!</button>
+              </li>
+            </ul>
+          </nav>
+        </div>
+      </header>
+      <section class="hero row">
+        <div class="hero__container container">
+          <article class="hero__content">
+            <h1 class="hero__header">Beautiful Free Nova template</h1>
+            <p class="hero__text">
+              A top notch premium quality template for your next web project.
+            </p>
+            <button class="hero__btn btn btn-green">Download FREE!</button>
+          </article>
+          <article class="hero__image">
+            <div class="hero__image-container">
+              <img
+                class="hero__img"
+                src="./images/screen.png"
+                alt="" />
+            </div>
+          </article>
+        </div>
+      </section>
+    </div>
+    <section
+      id="features"
+      class="features row">
+      <h2 class="features__header container">FEATURES</h2>
+      <div class="features__items container">
+        <div class="features__item">
```

Comment:
Jeśli mamy listę jakiś elementów (np. cech) to proponuję zrobić `ul > li`, gdzie `li` zawiera jeden konkretny element - będzie czytelniej.